scripts/buildtools.sh

```diff
@@ -53,5 +57,3 @@ if [ $is_183 -ne 0 ]; then
 else
     echo "CraftBukkit 1.8.3 installed; skipping BuildTools..."
 fi
-
-popd
```

Comment:
Why did you remove this line?

The `popd` is necessary to make sure the build script is in the right working directory after running BuildTools.

Reply:
Whoops, looks like I removed that on accident, let me fix it